microsoft-365/admin/misc/power-bi-in-your-organization.md

```diff
@@ -21,7 +21,7 @@ ROBOTS: NOINDEX
 description: "Learn about Power BI and how users in your organization can use this business analytics service."
 ---
 
-# Power BI in your organization
+# Using Power BI data in your organization
```

Comment:
@lhcole613 - I think you'll get dinged by a PubOps reviewer if you use a gerund in the heading, unless they changed this rule. They forced me to change a gerund once. Maybe change to "Overview of Power BI in your organization" if it's an issue.